dev/a11y_assessments/android/app/build.gradle

```diff
@@ -35,7 +35,13 @@ if (keystorePropertiesFile.exists()) {
 android {
     namespace "com.example.a11y_assessments"
     compileSdk flutter.compileSdkVersion
-    ndkVersion flutter.ndkVersion
+
+    // Flutter's CI installs the NDK at a non-standard path.
```

Comment:
Include a link (go or direct) that points to where that system install is configured.

Reply:
added